ADD_NEW_DATASET.md

```diff
@@ -23,7 +23,11 @@ This creates a copy of the code under your GitHub user account.
 	pip install -e ".[dev]"
 	```
 
-5. Prepare your tagging setup. You will need to clone the [tagging application repository](https://github.com/huggingface/datasets-tagging) and run the app with [Streamlit](https://www.streamlit.io/). You should also open the online form that will allow you to [create dataset cards](https://huggingface.co/datasets/card-creator/) in a browser window (courtesy of [Evrard t'Serstevens](https://huggingface.co/evrardts).)
+5. *Optionally*, prepare your tagging setup.
+  - Alternatively, you can use the [online Datasets Tagging application](https://huggingface.co/spaces/huggingface/datasets-tagging).
+  - Or clone the [Datasets Tagging application repository](https://github.com/huggingface/datasets-tagging) and run the app with [Streamlit](https://www.streamlit.io/).

```

Comment:
Maybe we should only advise to use the online tool ? WDYT ?

Reply:
Done